x/evm/types/contracts.go

```diff
@@ -1,14 +1,14 @@
-package types
+// Code generated by contractsgen. DO NOT EDIT.
```

Comment:
The permissions of this file changed to include execute?

Reply:
I do not understand this question

Reply:
the permission of this file was `644` before, now it's `755`. That seems unnecessary?

Reply:
fixed